website/docs/aws.md

```diff
@@ -1,105 +1,72 @@
 
-Before you begin, ensure you have the following tools installed:
-- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to provision AWS resources
-- [Eksctl](https://eksctl.io/installation/) (>= v0.191.0) to create and manage clusters on EKS
-- [Helm](https://helm.sh) to install this operator
-- [kubectl](https://kubernetes.io/docs/tasks/tools/) to view Kubernetes resources
+## Prerequisites
 
-## Create EKS Cluster
-If you do not already have an EKS cluster, run the following to create one:
+- An EKS cluster with KAITO workspace controller installed (see [Installation](installation))
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for managing AWS resources
+- [eksctl](https://eksctl.io/installation/) for EKS cluster management
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) configured to access your EKS cluster
 
-```bash
-cd ../.. #go back to main directory to use MAKE commands
-
-export AWS_CLUSTER_NAME=kaito-aws
-export AWS_REGION=us-west-2
-export AWS_PARTITION=aws
-export AWS_K8S_VERSION=1.30
-export KARPENTER_NAMESPACE=kube-system
-export AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
+## Understanding Auto-Provisioning on AWS
 
-make deploy-aws-cloudformation
-make create-eks-cluster
-```
-
-If you already have an EKS cluster, connect to it using
-```bash
-aws eks update-kubeconfig --name $CLUSTER_NAME --region $AWS_REGION
-```
+KAITO can use [Karpenter](https://karpenter.sh/) to automatically provision GPU nodes. This controller:
 
-## Install Karpenter Controller
-```bash
-make aws-karpenter-helm
-```
+- Creates new GPU nodes when workspaces require specific instance types
+- Supports various AWS GPU instances (g4, g5, p3, p4 series, etc.)
+- Manages node lifecycle based on workload demands
+- Integrates with AWS IAM for secure access
 
-## Install Workspace Controller
-```bash
-make aws-patch-install-helm
-```
+:::note
+Alternative: If you already have GPU nodes or manage them separately, use the preferred nodes approach in the [Quick Start](quick-start) instead.
+:::
 
-## Verify installation
-You can run the following commands to verify the installation of the controllers were successful.
+## Setup Auto-Provisioning
 
-Check status of the Helm chart installations.
+## Create EKS Cluster and install Karpenter
 
-```bash
-helm list -n default
-```
+Follow the instructions [here](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/) to create an EKS cluster and install Karpenter.
 
-Check status of `workspace`.
+Then update the KAITO workspace controller to the latest version that supports Karpenter integration:
 
 ```bash
-kubectl describe deploy kaito-workspace -n kaito-workspace
+helm update kaito-workspace --namespace kaito-workspace --set cloudProviderName=aws
```

Comment:
hmmm. this part is strange. Don't user already have KAITO installed in their cluster following previous steps.
KAITO supports karpenter from a very early release. We don't need this notice.

Reply:
this is updating to add the cloudProviderName

Reply:
I think it's still required. let's keep it here.